src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: 50%;
+    background-image: url("../images/star.svg");
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/c4e565b4-4c58-452c-beb5-b09a71768e0c)

Add a possibility to change the number of active stars